### PR TITLE
Formatting and attachments

### PR DESCRIPTION
**Pull request by ertygi**
Enable basic message formatting

### CODE REVIEW

File: src/Attachment.php [hunk 1]
```diff
@@ -125,6 +125,13 @@ class Attachment
      */
     protected $fields;
 
+    /**
+     * Message Formatting.
+     *
+     * @var array
+     */
+    protected $mrkdwn = [];
```
Comment (freekmurze) [inline src/Attachment.php]: Rename this to `useMarkdown`, let's store a boolean there with `false` as default

File: src/Attachment.php [hunk 2]
```diff
@@ -312,6 +319,22 @@ public function setAuthorName(string $authorName)
...
     }
 
+    /**
+     * Enable simple markup language.
+     *
+     * @param bool $setMrkdwn
+     *
+     * @return $this
+     */
+    public function setMarkdown(bool $setMrkdwn)
```
Comment (freekmurze) [inline src/Attachment.php]: Rename this to `useMarkdown`. Make it set the `useMarkdown` propery to `true`

File: src/Attachment.php [hunk 3]
```diff
@@ -511,6 +534,7 @@ public function toArray()
             'pretext'     => $this->preText,
             'color'       => $this->color,
             'footer'      => $this->footer,
+            'mrkdwn_in'   => $this->mrkdwn,
```
Comment (freekmurze) [inline src/Attachment.php]: Replace this by 

```
'mrkdwn_in'   => $this->userMarkdown ? ['text', 'pretext', 'footer'] : [];
```

File: src/Attachment.php (hunk 1 above)
Comment (ErtyGi) [reply src/Attachment.php]: Done.

File: src/Attachment.php (hunk 3 above)
Comment (ErtyGi) [reply src/Attachment.php]: Any other advice? Thank you @freekmurze

**Pull request merged**